crowd/components/OTPVerificationScreen.tsx

```
@@ -0,0 +1,345 @@
+              selectionColor={COLOR_SCHEME.PRIMARY}
+            />
+          ))}
+        </View>
+
+        <TouchableOpacity
+          style={[
+            styles.verifyButton,
+            ,
+            otpDigits.some((digit) => digit.trim() === "") &&
+              styles.disabledButton,
+          ]}
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)